GeeksCoreLibrary/Components/OrderProcess/Models/Constants.cs

```diff
@@ -173,10 +173,24 @@ public class Constants
 
         public const string PaymentMethodExternalNameProperty = "paymentmethodexternalname";
 
+        public const string MeasurementProtocolActive = "measurementprotocolactive";
```

Comment:
Kun je hier overal de suffix `Property` toevoegen, zoals bij vergelijkbare constanten? Dan is het consequent met de rest en is het duidelijker dat dit een propertynaam is en niet iets anders.